templates/layouts/one-column-overlay.html.twig

```diff
@@ -18,5 +18,6 @@
 {% else %}
   {% set hero_body = content.overlay %}
   {% set hero_image = content.main %}
+  {% set modifier_class = settings.extra_classes %}
   {% include "@decanter/components/hero/hero.twig" %}
```

Comment:
You'll likely want to unset the extra_classes or you will get double `class` attributes on the hero wrapper.

Reply:
ahh thats good Peer review.
@rebeccahongsf see https://github.com/SU-SWS/jumpstart_ui/blob/8.x-1.x/templates/components/quote/quote.html.twig#L13

Reply:
Ooo good to know! Thank you @sherakama and @pookmish 😄